src/main/resources/init_db.sql

```diff
@@ -0,0 +1,7 @@
+CREATE DATABASE `library`;
+USE library;
+CREATE TABLE `books` (
+    -> `id` BIGINT PRIMARY KEY NOT NULL AUTO_INCREMENT,
+    -> `title` VARCHAR(255),
+    -> `price` BIGINT
+    -> );
```

Comment:
```suggestion
    -> );
    
```
each file should end with one empty line